cmd/snap/main_test.go

```diff
@@ -93,10 +93,11 @@ func (s *BaseSnapSuite) Login(c *C) {
 }
 
 func (s *BaseSnapSuite) Logout(c *C) {
-	c.Assert(s.AuthFile == "", Equals, false)
-	err := os.Remove(s.AuthFile)
-	c.Assert(err, IsNil)
-	c.Assert(osutil.FileExists(s.AuthFile), Equals, false)
+	if osutil.FileExists(s.AuthFile) {
+		err := os.Remove(s.AuthFile)
+		c.Assert(err, IsNil)
+		c.Assert(osutil.FileExists(s.AuthFile), Equals, false)
```

Comment:
These three lines could be simply:
```
c.Assert(os.Remove(s.AuthFile), IsNil)
```
If Remove worked, we usually trust it did its job.